sdk/src/androidTest/java/com/mapbox/maps/CameraAnimationsPluginTest.kt

```diff
@@ -451,6 +460,123 @@ class CameraAnimationsPluginTest : BaseAnimationMapTest() {
     }
   }
 
+  @Test
+  fun testEaseToStartDelayCanceled() {
+    if (ignoreTestForGivenAbi()) {
+      return
+    }
+    val targetBearing1 = 5.0
+    val targetBearing2 = 10.0
+    val targetBearing3 = 15.0
+    val cameraOptions1 = CameraOptions.Builder().bearing(targetBearing1).build()
+    val cameraOptions2 = CameraOptions.Builder().bearing(targetBearing2).build()
+    val cameraOptions3 = CameraOptions.Builder().bearing(targetBearing3).build()
+    val expectedValues = mutableSetOf(targetBearing1, targetBearing3)
+    val updatedValues = mutableListOf<Double>()
+
+    val cameraAnimationPlugin = mapView.camera
+    val latch = CountDownLatch(2)
+    cameraAnimationPlugin.addCameraBearingChangeListener {
+      Logger.i(TAG, "onChanged $it")
+      updatedValues.add(it)
+      latch.countDown()
+    }
+
+    mainHandler.post {
+      cameraAnimationPlugin.easeTo(
+        cameraOptions1,
+        mapAnimationOptions {
+          duration(1)
+          startDelay(1)
+        }
+      )
+    }
+
+    mainHandler.postDelayed(
+      {
+        cameraAnimationPlugin.easeTo(
+          cameraOptions2,
+          mapAnimationOptions {
+            duration(1)
+            startDelay(3000)
+          }
+        )
+      },
+      100
+    )
+
+    // cameraOptions2 will be canceled since it's still waiting to start
+    mainHandler.postDelayed(
+      {
+        cameraAnimationPlugin.easeTo(
+          cameraOptions3,
+          mapAnimationOptions {
+            duration(1)
+            startDelay(0)
```

Comment:
let's remove delay at all here

Reply:
Set delay to 0 and remove it are two different scenarios. I'd like to keep it to verify delay equals 0 and workes well.